src/index.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: "Matt Cridland: Developer"
```

Comment:
Removed speech marks here, would this have fixed the colon issue in the second instance?